src/js/organisms/filter-form.js

```diff
@@ -51,7 +51,7 @@ module.exports = class FilterForm {
     });
 
     this.nextPageButton.addEventListener('click', e => this.nextPage(e));
-    this.form.addEventListener('input', () => this.onFormInput());
+    this.form.addEventListener('change', () => this.onFormInput());
```

Comment:
I didn’t know [older browsers and pre-Blink Edge don’t trigger `input` on checkboxes][1], nice catch!

[1]: https://caniuse.com/#feat=input-event

Reply:
Me neither. Took a while to find!